web-common/src/features/dashboards/time-controls/DefaultTimeRangeMenuItem.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import { humaniseISODuration } from "@rilldata/web-common/lib/time/ranges/iso-ranges";
+  import MenuItem from "../../../components/menu/core/MenuItem.svelte";
+
+  export let selected = false;
+  export let isoDuration: string;
+</script>
+
+<MenuItem animateSelect={false} on:select>
```

Comment:
If we have the right intermediate value, `animateSelect` being `true` makes sense and is consistent with the other menu items.